docs/usage-with-javascript.md

```diff
@@ -17,8 +18,8 @@ typeorm.createConnection({
     database: "test",
     synchronize: true,
     entities: [
-        require("./entity/Post"),
-        require("./entity/Category")
+        new EntitySchema(require("./entity/Post")),
+        new EntitySchema(require("./entity/Category"))
```

Comment:
Are you sure this is necessary?

Reply:
Yes. 

The items in the entities array should be either a constructor function with ```@Entity()``` metadata, or an instanceof ```EntitySchema```, or string that is a file glob path to modules that export one of those things. Since this is an example using plain javascript, it needs to be an EntitySchema. This is the same technique for defining entities as described in [Separating Entity Definition](https://github.com/typeorm/typeorm/blob/master/docs/separating-entity-definition.md).

If if you try to run the example code as shown, it will throw a RepositoryNotFoundError. The code in the example repo linked at the bottom of this document works correctly. This updates the example code on the document itself to match the code in the repo.

Reply:
Right, didn't read the whole example.  Sorry :)

Can we instead create the entity schemas in the entity objects instead?

Reply:
OK, I like that better. I think the example repo does it in the entities array for importing from JSON, but that's not the case here. I will revise the pull request. Thanks!

Reply:
So I used ```var EntitySchema=require("typeorm").EntitySchema``` in the schema files. Is that ok?

(I assume it's using var for compatibility with ES5, but I don't know if we really need to do that anymore.)